### ISSUE TITLE
Resurrect pop-up window problems with keyboard navigation

### ISSUE THREAD

**Issue by pabouk** [arantius/resurrect-pages#1]
In Firefox 22.0 on Linux with Resurrect Pages 2.0.7 I noticed that the buttons representing the individual web mirrors in the Resurrect pop-up window do not react properly to keyboard navigation. They can be properly selected using Tab and Shift+Tab but they cannot be activated. Space only animates the button press but that is all. Enter does not do anything visible.
